cdc/http_status.go

```diff
@@ -54,6 +57,11 @@ func (s *Server) startStatusHTTP() error {
 
 	serverMux.HandleFunc("/admin/log", handleAdminLogLevel)
 
+	if util.FailpointBuild {
+		// `http.StripPrefix` is needed because `failpoint.HttpHandler` assumes that it handles the prefix `/`.
+		serverMux.Handle("/fail/", http.StripPrefix("/fail", &failpoint.HttpHandler{}))
```

Comment:
```suggestion
		serverMux.Handle("/debug/fail/", http.StripPrefix("/debug/fail", &failpoint.HttpHandler{}))
```

I prefer to add a prefix /debug.